src/data_management/scenarios/Scenarios.css

```diff
@@ -149,6 +149,21 @@
     color: #818181;
 }
 
+.NoResults {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 300px;
+  align-items: center;
+}
+.NoResults h5 {
+  color: #C1C1C1;
+  font-weight: 300;
+  margin: 30px;
+}
```

Comment:
I can see that this piece of CSS styling is used multiple times in the whole App (i.e. the alarm notifications, raster, wms layers, contacts, templates, groups ...). It would be nice if we can reuse them across the whole app.

Reply:
I agree. I will put this in the refactoring ticket. :+1: